### PR TITLE
feat: propogate descriptions in the ui if they are simply pass throug…

### PR DESCRIPTION
**Pull request by tobymao**
…h and don't have multiple sources

### CODE REVIEW

File: sqlmesh/core/lineage.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
```
Comment (izeigerman) [inline sqlmesh/core/lineage.py:1]: Is this only used for web? Should we move this into the `web` package?

File: sqlmesh/core/lineage.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                continue
+
+            table = d.expression.find(exp.Table)
+            if table:
+                name = normalize_model_name(table, default_catalog=context.default_catalog, dialect=model.dialect)
+                parents[name].add(exp.to_column(d.name).name)
+    return dict(parents)
+
+
+def column_description(context: Context, model_name: str, column: str) -> t.Optional[str]:
```
Comment (izeigerman) [inline sqlmesh/core/lineage.py:48]: We'll need some tests for this

File: sqlmesh/core/lineage.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    dependencies = column_dependencies(context, model_name, column)
+
+    if len(dependencies) != 1:
+        return None
+
+    for parent, columns in dependencies.items():
+        if len(columns) != 1:
+            return None
+
+        for column in columns:
```
Comment (izeigerman) [inline sqlmesh/core/lineage.py:63]: [Nit] Seems like an unnecessary level of nestedness

File: sqlmesh/cli/main.py [hunk 4]
```diff
@@ -591,6 +591,7 @@ def ui(
         port=port,
         log_level="info",
         timeout_keep_alive=300,
+        reload=True,
```
Comment (vchan) [inline sqlmesh/cli/main.py:594]: I think we should remove this.

Comment (tobymao) [reply sqlmesh/cli/main.py:594]: yea i will

**Comment by tobymao** [comment]
@vchan merging this after tests pass

**Pull request merged**